debian-8/post_install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+curl https://raw.githubusercontent.com/viglesiasce/cloud-images/master/utils/rc.local > /etc/rc.local
```

Comment:
we can remove it. this is actually sort of a hack -- that 169 addr is the aws metadata server.